experimental/dds/tree2/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -109,11 +109,27 @@ function invertMark<TNodeChange>(
 			return [mark];
 		}
 		case "Insert": {
-			const inverse = withNodeChange(
-				{ type: "Delete", count: mark.content.length },
-				invertNodeChange(mark.changes, inputIndex, invertChild),
-			);
-			return [inverse];
+			if (mark.detachedBy !== undefined) {
+				assert(revision !== undefined, "Unable to revert to undefined revision");
+				return [
```

Comment:
It looks like this is saying that the inverse of a transient insert is an active revive. Are you sure this is right?